resources/views/home.blade.php

```diff
@@ -15,38 +15,63 @@
                     @endif
                     <table class="products table">
                         <tr>
-                            <td>
+                            <th></th>
+                            <th>
                                 <strong>Thumbnail</strong>
-                            </td>
-                            <td>
+                            </th>
+                            <th>
                                 <strong>Product</strong>
-                            </td>
-                            <td>
+                            </th>
+                            <th class="table-brand">
                                 <strong>Brand</strong>
-                            </td>
-                            <td>
+                            </th>
+                            <th hidden>
```

Comment:
hidden?

Reply:
Although not widely used, actually a valid attribute.
The question should be: why render the element if we're going to hide it anyway?

Reply:
It was for later if there is a Admin or something like that ,but I think I can remove it for now.